### DISCUSSION TITLE
How can I fix CPU utilization and core clock not showing any value?

### DISCUSSION THREAD

**Discussion by trefeon** [BartoszCichecki/LenovoLegionToolkit#1176] [Q&A] [answered]
![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/95156714/72c7f9f8-8816-487b-8685-0a40e6809eac)

**Comment by BartoszCichecki** [comment, marked as answer]
It's mostly an issue with Windows performance counters. You can try resetting them.
